articles/azure-functions/functions-bindings-kusto-managed-identity.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Connect a function app to Kusto with managed identity - Preview

```

Comment:
Connect an Azure Functions app to Azure Data Explorer with managed identity (Preview)